packages/helpermodules/setdata.py

```diff
@@ -843,6 +835,32 @@ def process_general_topic(self, msg: mqtt.MQTTMessage):
         except Exception:
             log.exception(f"Fehler im setdata-Modul: Topic {msg.topic}, Value: {msg.payload}")
 
+    def process_io_topic(self, msg: mqtt.MQTTMessage):
+        """ Handler für die Allgemeinen-Topics
+
+         Parameters
+        ----------
+
+        msg:
+            enthält Topic und Payload
+        """
+        try:
+            if "config" in msg.topic:
+                self._validate_value(msg, "json")
+            elif ("get/digital_input" in msg.topic or
+                  "get/digital_output" in msg.topic or
+                  "get/analog_input" in msg.topic or
+                  "get/analog_output" in msg.topic):
```

Comment:
Der Topic-Pfad "get" ist für die Ausgänge nicht wirklich passend.
Analog zu den Ladepunkten sollte "get" für Stati (hier Eingänge) und "set" für gesetzte Werte (hier Ausgänge) verwendet werden. Diese Änderung zieht zwangsläufig weitere Korrekturen nach sich, die ich in dem Review nicht zu 100% kommentieren kann.
```suggestion
            elif ("get/digital_input" in msg.topic or
                  "set/digital_output" in msg.topic or
                  "get/analog_input" in msg.topic or
                  "set/analog_output" in msg.topic):
```